package.json

```diff
@@ -59,7 +59,7 @@
   "scripts": {
     "_comment": "ONLY RUN BUILD IF YOUR SYSTEM HAS 8GB OR MORE OF RAM",
     "build": "node seed && node --max_old_space_size=7168 ./node_modules/.bin/gatsby build",
-    "dev": "node seed && node --max_old_space_size=7168 ./node_modules/.bin/gatsby develop",
+    "dev": "gatsby develop",
```

Comment:
@Bouncey Whoops! Wasn't paying enough attention when I made this change. Has now been fixed